app/docker/components/container-quick-actions/containerQuickActions.js

```diff
@@ -0,0 +1,10 @@
+angular.module('portainer.docker').component('containerQuickActions', {
+  templateUrl: 'app/docker/components/container-quick-actions/containerQuickActions.html',
+  bindings: {
+    containerId: '=',
+    nodeName: '=',
+    status: '=',
+    settings: '=',
```

Comment:
I'd rather see this binding being called `state` instead of `settings`.

```suggestion
    state: '=',
```